tests/unit/matrix/test_cubemeasure.py

```diff
@@ -1642,14 +1784,30 @@ def it_knows_its_table_bases(self, request):
             [7.6, 7.6, 7.6],
         ]
 
-    def it_knows_its_table_margin(self, raw_weighted_counts):
+    def it_knows_its_table_margin_x_nonarray(
```

Comment:
for example here you are testing the case of MR X CAT so it can be `it_knows_table_margin_for_mr_x_cat`